Fix missing header inclusion

INT_MAX is defined in <limits.h>; which isn't necessarily included by other of
the used headers, so in some libc implementations this causes a build error.

diff --git a/lib/zip_crypto_openssl.c.orig b/lib/zip_crypto_openssl.c
index 1cb004798b1c..74c1d87bc36c 100644
--- a/lib/zip_crypto_openssl.c
+++ b/lib/zip_crypto_openssl.c
@@ -31,6 +31,7 @@
   IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
 
+#include <limits.h>
 #include <stdlib.h>
 
 #include "zipint.h"
